README.md

```diff
@@ -222,3 +222,63 @@ logger.error('error message', { error: new Error('error') });
 ```
 
 </details>
+
+## Monday API Client
+
+<details>
+<summary>Querying the monday.com GraphQL API seamlessly on behalf of the connected user, or using a provided API token.</summary>
+
+#### initialize
+
+```typescript
+import { MondayApiClient } from '@mondaycom/apps-sdk';
+
+const mondayClient = new MondayApiClient();
+mondayClient.setToken(token);
+mondayClient.setApiVersion('2023-10');
```

Comment:
Question out of gap in knowledge. Is this the best practice we dictate to our developers? to hardcode the api version? there is no env variable or something we suggest using/creating to manage this?